src/lib/select/select.spec.ts

```diff
@@ -237,6 +237,27 @@ describe('MdSelect', () => {
       expect(fixture.componentInstance.select.selected).not.toBeDefined();
     });
 
+
+    it('should emit an event when the selected option has changed', () => {
```

Comment:
This should have its own test component that doesn't use a form module directive